docs/source/en/quicktour.mdx

```diff
@@ -14,53 +14,22 @@ specific language governing permissions and limitations under the License.
 
 [[open-in-colab]]
 
-Get up and running with 🤗 Transformers! Start using the [`pipeline`] for rapid inference, and quickly load a pretrained model and tokenizer with an [AutoClass](./model_doc/auto) to solve your text, vision or audio task.
+The 🤗 Transformers library is built around four major concepts to maximize user-friendliness and flexibility:
 
-<Tip>
-
-All code examples presented in the documentation have a toggle on the top left for PyTorch and TensorFlow. If
-not, the code is expected to work for both backends without any change.
-
-</Tip>
-
-## Pipeline
-
-[`pipeline`] is the easiest way to use a pretrained model for a given task.
-
-<Youtube id="tiZFewofSLM"/>
-
-The [`pipeline`] supports many common tasks out-of-the-box:
-
-**Text**:
-* Sentiment analysis: classify the polarity of a given text.
-* Text generation (in English): generate text from a given input.
-* Name entity recognition (NER): label each word with the entity it represents (person, date, location, etc.).
-* Question answering: extract the answer from the context, given some context and a question.
-* Fill-mask: fill in the blank given a text with masked words.
-* Summarization: generate a summary of a long sequence of text or document.
-* Translation: translate text into another language.
-* Feature extraction: create a tensor representation of the text.
-
-**Image**:
-* Image classification: classify an image.
-* Image segmentation: classify every pixel in an image.
-* Object detection: detect objects within an image.
-
-**Audio**:
-* Audio classification: assign a label to a given segment of audio.
-* Automatic speech recognition (ASR): transcribe audio data into text.
-
-<Tip>
```

Comment:
I think this introduction in different tasks was quite useful! As a user I feel like I wouldn't necessarily come to the quicktour to learn how to use the library deeply, but rather with a need and a task I'd want to solve. In this case, then showcasing what is supported straight away would be helpful.

Reply:
Ok, I think Sylvain also advocated for showcasing what all is supported so I'll keep it. Going to experiment a bit with presenting the tasks in a table :)